_config.yml

```diff
@@ -2,6 +2,10 @@
 title: Computing @ NYJC
 timezone: UTC
 collections:
+  two-column: 
```

Comment:
This change to the config file does not appear to be necessary. It is used to tell jekyll that there are markdown files in a particular folder to be parsed and made available in the static site (see https://jekyllrb.com/docs/collections/).

We do not have a "two-column" folder to be parsed, so this change should be removed.